src/transformStateWithClones.js

```diff
@@ -6,8 +6,37 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arrayStateConditions = [];
+  let stateNew = { ...state };
```

Comment:
Consider using a more descriptive name for the object that will hold the new state, such as `newState` or `currentState`, to improve code readability.